make/mt-config/mt-tls-wolfssl.mk

```diff
@@ -17,3 +17,9 @@ XI_CONFIG_FLAGS += -DXI_TLS_OCSP_STAPLING
 # XI_CONFIG_FLAGS += -DXI_TLS_OCSP
 
 XI_CONFIG_FLAGS += -DXI_TLS_LIB_WOLFSSL
+
+XI_CONFIG_FLAGS += -DWC_RSA_BLINDING
+
+XI_CONFIG_FLAGS += -DNO_FILESYSTEM
+
+XI_CONFIG_FLAGS += -DWOLFSSL_NO_SOCK
```

Comment:
New WolfSSL preprocessor definitions required for compilation.   This is for  our makefile environment.